go-controller/pkg/networkmanager/nad_controller_test.go

```diff
@@ -588,6 +591,19 @@ func TestSyncAll(t *testing.T) {
 			},
 			syncAllError: ErrNetworkControllerTopologyNotManaged,
 		},
+		{
+			name: "invalid network reference with erroneous subnet entry doesn't block sync",
+			testNADs: []TestNAD{
+				{
+					name:    "test/nad1",
+					netconf: network_A,
+				},
+				{
+					name:    "test2/nad2",
+					netconf: &network_A_Copy_InvalidSubnet,
```

Comment:
ah the fields need to be exactly the same here cause its two different NADs referecing same network..?

Reply:
ya, same network name but different network subnets.